.github/workflows/phar-creation.yml

```diff
@@ -4,22 +4,20 @@ name: "Building Require-Checker phar"
 
 on:
     pull_request:
+#    release:
+#      types: [published]
```

Comment:
I suppose it should still operate on releases?

Reply:
yes, but currently I have no idea how to test it

Reply:
it should run only on published releases now